XML/Snippet.sublime-syntax

```diff
@@ -0,0 +1,54 @@
+%YAML 1.2
+---
+# See http://www.sublimetext.com/docs/3/syntax.html
+file_extensions:
+  - sublime-snippet
+name: Sublime Text Snippet (XML)
+scope: text.xml.sublime-snippet
```

Comment:
Note that the `new_snippet` command in `Packages/Default/new_templates.py` currently references the normal XML syntax, and this should be changed in the ST3 codebase after merging.

    v.assign_syntax('Packages/XML/XML.sublime-syntax')